styles/perplexity/catppuccin.user.css

```diff
@@ -0,0 +1,165 @@
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    .bg-offset,
+    .bg-offsetDark,
+    :is(:where(.dark) .dark\:\!bg-offsetDark) {
+      background-color: @mantle !important;
+    }
+    1
+    .bg-offset:hover,
+    :is(:where(.dark) .md\:dark\:hover\:bg-offsetDark:hover) {
+        background-color: @mantle !important;
+    }
+
+    :is(:where(.dark) .dark\:md\:hover\:bg-offsetPlusDark:hover) {
+        background-color: @crust !important;
+    }
+
+    .text-textMainDark,
+    :is(:where(.dark) .dark\:text-textMainDark) {
+      color: @text;
+    }
+
+    :is(:where(.dark) .dark\:bg-offsetDark) {
+      background: @surface0;
+    }
+
+    :is(:where(.dark) .dark\:bg-backgroundDark) {
+      background-color: @base;
+    }
+
+    // buttons + toggle circle
+    :is(:where(.dark) .dark\:bg-offsetPlusDark) {
+      background: @surface1;
+    }
+
+    // off button
+    :is(:where(.dark) .dark\:bg-idleDark) {
+      background: @surface1;
+    }
+
+    // borders
+    :is(:where(.dark) .dark\:border-borderMainDark) {
+      border-color: @surface1 !important;
+    }
+
+    // ring border
+    :is(:where(.dark) .dark\:ring-borderMainDark) {
+      --tw-ring-color: rgb(#rgbify(@crust)[]/var(--tw-ring-opacity))
+    }
+
+    .border-borderMainDark\/80, .divide-borderMain\/60 {
+      border-color: @surface1;
+    }
+
+    // code block
+    .ePundr, [style^="font-size: inherit; font-family: inherit; background: rgb(45, 45, 45); color: rgb(204, 204, 204); border-radius: 3px; display: flex; line-height: 1.42857; overflow-x: auto; white-space: pre;"] {
+      background: @surface0  !important;
+      color: @text !important;
+    }
+
+    // darker text on light bg
+    :is(:where(.dark) .dark\:bg-textMainDark),
+    :is(:where(.dark) .dark\:text-backgroundDark) {
+      color: @crust;
+    }
+
+    // accent color buttons
+    :is(:where(.dark) .dark\:bg-superDark) {
+      background: @accent-color;
+    }
+
+    // subtexts
+    .text-textOff,
+    :is(:where(.dark) .dark\:text-textOffDark) {
+      color: @subtext0;
+    }
+
+    // prose
+    .prose,
+    :is(:where(.dark) .dark\:prose-invert) {
+      --tw-prose-body: @text;
+      --tw-prose-headings: @text;
+      --tw-prose-lead: @text;
+      --tw-prose-links: @blue;
+      --tw-prose-bold: @text;
+      --tw-prose-counters: @text;
+      --tw-prose-bullets: @text;
+      --tw-prose-hr: @text;
+      --tw-prose-quotes: @text;
+      --tw-prose-quote-borders: @text;
+      --tw-prose-captions: @text;
+      --tw-prose-kbd: @text;
+      --tw-prose-kbd-shadows: @text;
+      --tw-prose-code: @text;
+      --tw-prose-pre-code: @text;
+      --tw-prose-pre-bg: @base;
+      --tw-prose-th-borders: @text;
+      --tw-prose-td-borders: @text;
+    }
+  }
+
+  :root[class="dark"] {
```

Comment:
```suggestion
  :root.dark {
```

Reply:
clearly i was not thinking very much when i did this lol